.github/workflows/check-http-status-code.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+URL=$1; shift
+EXPECTED_STATUS=$1; shift
+
+# STATUSCODE=$(docker run --network container:nginx curlimages/curl --silent -v $* --output /dev/stderr --write-out "%{http_code}" "$URL")
+STATUSCODE=$(curl --insecure --silent -v $* --output /dev/stderr --write-out "%{http_code}" "$URL")
+
+if [ $? -ne 0 ]; then
+  echo "❌ Curl failed"
```

Comment:
Should we print the URL that failed here?

Reply:
it is printed by curl with the `-v` (verbose) argument